app/src/main/res/layout/widget_entry.xml

```diff
@@ -4,29 +4,34 @@
     android:layout_height="wrap_content"
     android:layout_width="match_parent"
     android:orientation="horizontal"
-    android:padding="4dp">
+    android:padding="@dimen/widget_note_list_entry_layout_padding">
 
     <TextView
         android:id="@+id/widget_entry_content_tv"
-        android:layout_width="wrap_content"
+        android:layout_width="@dimen/widget_note_list_entry_content_width"
         android:layout_height="match_parent"
-        android:paddingLeft="6dp"
+        android:paddingStart="@dimen/widget_note_list_entry_padding_start"
+        android:paddingEnd="@dimen/widget_note_list_entry_padding_end"
```

Comment:
Why do use different dimensions for padding left and start (respective right and end)? These values should be the same and therefore the dimension keys, too (if used at all, see other comment). I don't think, that the file `values-v17/dimens.xml` is needed.